pulp/pulp.py

```diff
@@ -1302,15 +1755,14 @@ def __init__(self, name="NoName", sense=const.LpMinimize):
         if " " in name:
             warnings.warn("Spaces are not permitted in the name. Converted to '_'")
             name = name.replace(" ", "_")
-        self.objective = None
-        self.constraints = _DICT_TYPE()
+        self.constraints = dict()
         self.name = name
         self.sense = sense
         self.sos1 = {}
         self.sos2 = {}
         self.status = const.LpStatusNotSolved
         self.sol_status = const.LpSolutionNoSolutionFound
-        self.noOverlap = 1
+        self.noOverlap = False
```

Comment:
this changes the code. Shouldn't it be True?

Reply:
Sorry, it should be True.